configure

```diff
@@ -9647,1253 +9647,31 @@ tcl_version=8.5.18
-					;;
-					--*=* | -*=*)
-							mp_arg_key=$($as_echo "$mp_arg" | sed -E 's/^--?([^=]+)=.*$/\1/')
-							mp_list_contains_present=
-	eval "mp_list_contains_list=\"$mp_sub_configure_keys\""
-	for mp_list_contains_entry in $mp_list_contains_list; do
-		test -z "$mp_list_contains_entry" && continue
-		if test "x$mp_arg_key" = "x$mp_list_contains_entry"; then
-			mp_list_contains_present=1
-			break
-		fi
-	done
-	if test "x$mp_list_contains_present" = "x"; then
-		:
-		else
-									case $mp_arg in
-		*\'*)
-			mp_arg=$($as_echo "$mp_arg" | sed "s/'/'\\\\\\\\''/g")
-		;;
-	esac
-	mp_arg="'$mp_arg'"
-	as_fn_append mp_sub_configure_args " $mp_arg"
-
-
-	fi
-
-					;;
-					*)
-						# always pass positional arguments
-								case $mp_arg in
-		*\'*)
-			mp_arg=$($as_echo "$mp_arg" | sed "s/'/'\\\\\\\\''/g")
-		;;
-	esac
-	mp_arg="'$mp_arg'"
-	as_fn_append mp_sub_configure_args " $mp_arg"
-
-					;;
-				esac
-			done
-
-			# Now prepend some arguments that should always be present unless
-			# overriden, such as --prefix, --silent, --disable-option-checking,
-			# --cache-file, --srcdir
-			# Pass --prefix unless already given
-			mp_arg_key=prefix
-				mp_list_contains_present=
-	eval "mp_list_contains_list=\"$mp_sub_configure_args\""
-	for mp_list_contains_entry in $mp_list_contains_list; do
-		test -z "$mp_list_contains_entry" && continue
-		if test "x$mp_arg_key" = "x$mp_list_contains_entry"; then
-			mp_list_contains_present=1
-			break
-		fi
-	done
-	if test "x$mp_list_contains_present" = "x"; then
-		:
-		else
-				mp_arg="--prefix=$prefix"
-						case $mp_arg in
-		*\'*)
-			mp_arg=$($as_echo "$mp_arg" | sed "s/'/'\\\\\\\\''/g")
-		;;
-	esac
-	mp_arg="'$mp_arg'"
-			mp_fn_var_prepend mp_sub_configure_args "$mp_arg "
-
-
-
-	fi
-
-
-			# Pass --silent
-			if test "$silent" = yes; then
-				mp_arg="--silent"
-						case $mp_arg in
-		*\'*)
-			mp_arg=$($as_echo "$mp_arg" | sed "s/'/'\\\\\\\\''/g")
-		;;
-	esac
-	mp_arg="'$mp_arg'"
-			mp_fn_var_prepend mp_sub_configure_args "$mp_arg "
-
-
-			fi
-
-			# Always prepend --disable-option-checking to silence warnings, since
-			# different subdirs can have different --enable and --with options.
-			mp_arg="--disable-option-checking"
-					case $mp_arg in
-		*\'*)
-			mp_arg=$($as_echo "$mp_arg" | sed "s/'/'\\\\\\\\''/g")
-		;;
-	esac
-	mp_arg="'$mp_arg'"
-			mp_fn_var_prepend mp_sub_configure_args "$mp_arg "
-
-
-
-			# Make the cache file name correct relative to the subdirectory.
-			case $cache_file in
-				[\\/]* | ?:[\\/]* )
-					mp_sub_cache_file=$cache_file
-				;;
-				*) # Relative name.
-					mp_sub_cache_file=$ac_top_build_prefix$cache_file
-				;;
-			esac
-			mp_arg="--cache-file=$mp_sub_cache_file"
-					case $mp_arg in
-		*\'*)
-			mp_arg=$($as_echo "$mp_arg" | sed "s/'/'\\\\\\\\''/g")
-		;;
-	esac
-	mp_arg="'$mp_arg'"
-			mp_fn_var_prepend mp_sub_configure_args "$mp_arg "
+MP_CONFIG_SUBMODULE(vendor/tcl, unix, --prefix=${macports_libexec_dir} --enable-threads)
```

Comment:
You forgot to define `MP_CONFIG_SUBMODULE` so it shows up unexpanded in the generated configure script.